run_dir/design/base.html

```diff
@@ -15,7 +15,7 @@
 <script src="/static/js/moment.min.js"></script>
 
 <!-- Le styles -->
-<link type="text/css" rel="stylesheet" href="/static/nvd3/src/nv.d3.css">
+<!-- <link type="text/css" rel="stylesheet" href="/static/nvd3/src/nv.d3.css"> -->
```

Comment:
Unless I'm mistaken, base.html should be extended by every other page out there. Removing a css from this might make some other pages behave in an odd way. Did you check that ?